cogs5e/utils/gameutils.py

```diff
@@ -16,3 +32,84 @@ async def send_hp_result(ctx, caster, delta=None):
         await caster.message_controller(ctx, f"{caster.name}'s HP: {caster.hp_str(True)}{deltaend}")
     else:
         await ctx.send(f"{caster.name}: {caster.hp_str()}{deltaend}")
+
+
+async def send_current_coin(ctx, character, coin="All"):
+    """
+    Sends the current contents of the CoinPurse
+    """
+    cp_display_embed = EmbedWithColor()
+    cp_display_embed.colour = 15844367
+    cp_display_embed.set_footer(text=f"For help managing your coins, use !game coinpurse")
+    cp_display_embed.set_thumbnail(url="https://www.dndbeyond.com/attachments/thumbnails/3/929/650/358/scag01-04.png")
+    cp_display_embed.add_field(name="Total Value in Purse", value=f"{CoinTypes['gp']['icon']} gp: {character.coinpurse.total}", inline=False)
```

Comment:
```suggestion
    cp_display_embed.add_field(name="Total Value", value=f"{character.coinpurse.total} gp", inline=False)
```
imo since this isn't the actual currency, we shouldn't show the coin itself